assets/js/helpers/metrics.js

```diff
@@ -20,3 +20,13 @@ export function trackEvent(category, action, label) {
         ga('send', payload);
     }
 }
+
+/**
+ * Tag recording
+ * Helper to add tag(s) to the current session's Hotjar recording
+ */
+export function tagHotjarRecording(tags) {
+    if (window.hj && tags) {
```

Comment:
Not sure we need the `&& tags` check, we're always passing this in as the function argument.

Reply:
True, I guess it's just a sanity check, but since it's only one parameter I guess it's safe enough.